ci/infra/testrunner/skuba/skuba.py

```diff
@@ -170,6 +170,21 @@ def _run_skuba(self, cmd, cwd=None):
         if cwd is None:
             cwd = self.cwd
 
+        if verbosity is None:
+           verbosity = self.conf.skuba.verbosity  # noqa: E111
+        try:
+            v = int(verbosity)
+        except ValueError:
+            msg = f"verbosity '{verbosity}' is not an int"
+            logger.error(msg)
+            raise Exception(msg)
```

Comment:
I think it makes a lot more sense to just default to 2 here like you had it in the first place, but I was late to the party..

Reply:
Whoops; I forgot to change that to `logger.exception()` and just inline the `msg`.

Reply:
I switched it to raise a more descriptive ValueError exception.  The vars file contains a default value of "2", and the class has a default of 2.  If the config file is old and the var isn't present, it'll use "2".  If the config file is new and the value is something other than 2, then the user changed it intentionally _away from_ 2 and we should notify them that they chose an invalid value.  It might make sense to check for an empty string and use that to suppress adding the `-v`; defaulting to the program's default verbosity level - but that seems an unlikely use case.